pkg/service/model_config.go

```
@@ -116,11 +135,36 @@ const (
 )
 
 type Action struct {
+	Delay      *time.Duration // e.g. "12h" or "10s"
 	Copy       *Copy
 	Correction *Correction
 	Return     *Return
 }
 
+func (a *Action) Validate() bool {
+	// Delay is only valid for Return and Correction
+	if a.Delay != nil && a.Copy != nil {
+		return false
```

Do we want to return an error describing the problem?